components/aws/sagemaker/tests/unit_tests/tests/test_utils.py

```diff
@@ -1,10 +1,45 @@
-def check_empty_string_values(obj):
-    obj_has_empty_string = 0
-    if type(obj) is dict:
-        for k,v in obj.items():
-            if type(v) is str and v == '':
-                print(k + '- has empty string value')
-                obj_has_empty_string = 1
-            elif type(v) is dict:
-                check_empty_string_values(v)
+import unittest
 
+from unittest.mock import patch, call, Mock, MagicMock, mock_open
+from botocore.exceptions import ClientError
+
+from common import _utils
+
+
+# Test common util functions used across the components
+class UtilsTestCase(unittest.TestCase):
+    def test_cw_logging_successfully(self):
+        mock_cw_client = MagicMock()
+        mock_cw_client.describe_log_streams = MagicMock(return_value={'logStreams': [{'logStreamName': 'logStream1'},
```

Comment:
All children of a `MagicMock` object are also MagicMocks by default. So here you can simplify by saying `mock_cw_client.describe_log_streams.return_value = ...`

Reply:
oh cool 
will change